src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -37,20 +39,23 @@ trait WorkflowComponent {
   import driver.api._
 
   class WorkflowTable(tag: Tag) extends Table[WorkflowRecord](tag, "WORKFLOW") {
-    def id = column[UUID]("ID", O.PrimaryKey)
+    def id = column[Long]("ID", O.PrimaryKey, O.AutoInc)
+    def externalid = column[String]("EXTERNAL_ID")
```

Comment:
Important change here: we no longer use the cromwell workflow id as our primary key. The initial impetus to change was because the mock remote server in our tests always serves the same workflow id. However, this is a better design because one day cromwell may decide to return us a non-unique (to us) id then boom. Or perhaps there is some use case where this is desirable.